src/com/puppetlabs/puppetdb/cli/anonymize.clj

```diff
@@ -18,161 +18,108 @@
 
 (def anon-profiles
   ^{:doc "Hard coded rule engine profiles indexed by profile name"}
-  {
-    "full" {
-      ;; Full anonymization means anonymize everything
-      "rules" {}
-    }
-    "moderate" {
-      "rules" {
-        "type" [
-          ;; Leave the core type names alone
-          {"context" {"type" [
-            "Augeas" "Computer" "Cron" "Exec" "File" "Filebucket" "Group" "Host"
-            "Interface" "K5login" "Macauthorization" "Mailalias" "Mcx" "Mount"
-            "Notify" "Package" "Resources" "Router" "Schedule" "Schedule_task"
-            "Selboolean" "Selmodule" "Service" "Ssh_authorized_key" "Sshkey" "Stage"
-            "Tidy" "User" "Vlan" "Yumrepo" "Zfs" "Zone" "Zpool"]}
-           "anonymize" false}
-          {"context" {"type" "/^Nagios_/"} "anonymize" false}
+  {"full" {;; Full anonymization means anonymize everything
+           "rules" {}}
+   "moderate" {"rules" {"type" [;; Leave the core type names alone
```

Comment:
That's contrary to standard lisp commenting rules too. `;;` indicates a comment relating to the next line, while `;` is used for an end-of-line comment